package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "setup-dbs": "psql -f ./db/setup.sql",
     "seed": "node ./db/seeds/run-seed.js",
-    "test": "npm run setup-dbs && jest",
+    "test": "jest",
```

Comment:
The setup should keep happening at any test, you may alter the database structure and this can make future test fail

Reply:
You're right - I thought I had to change it but David informed me it wasn't necessary in the last PR and I reverted back to the old "test" script.